padatious/intent_container.py

```diff
@@ -27,10 +34,13 @@ class IntentContainer(object):
         cache_dir (str): Place to put all saved neural networks
     """
     def __init__(self, cache_dir):
+        if not isdir(cache_dir):
+            mkdir(cache_dir)
```

Comment:
would it be prudent to use `os.makedirs()` in case multiple levels needs to be created?

Reply:
Good idea! Changed.